backend/fms_core/viewsets/sample_next_step.py

```diff
@@ -399,14 +399,34 @@ def labwork_step_info(self, request: HttpRequest, *args, **kwargs):
 
         grouped_step_samples = self.filter_queryset(self.get_queryset())
         # Get all samples on the steps with the grouping field
-        grouped_step_samples = grouped_step_samples.filter(step__id__exact=step_id).values_list("sample_id", grouping_column, "ordering_container_barcode", "ordering_container_coordinates")
+        grouped_step_samples = grouped_step_samples.filter(step__id__exact=step_id) \
+            .annotate(project_name=F("sample_next_step_by_study__study__project__name")) \
+            .annotate(sample_name=F("sample__name")) \
+            .values_list(
+                "sample_id",
+                "sample_name",
+                "project_name",
```

Comment:
This is not slowing the query too much on the samples tab ?

Reply:
I tried dozens of container, and it took a few seconds to load of the containers.
But yeah it would be nice to optimize this when we go big.